.github/ISSUE_TEMPLATE/regression.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Regression
+about: Report a regression in Rust.
+labels: C-bug
+---
+<!--
+Thank you for filing a bug report! 🐛 Please provide a short summary of the bug,
+along with any information you feel relevant to replicating the bug.
+-->
+
+### Code
+
+I tried this code:
+
+```rust
+<code>
+```
+I expected to see this happen: *explanation*
+
+Instead, this happened: *explanation*
+
+### Version it worked on
+
+<!--
+Provide the most recent version this worked on, for example:
+
+It most recently worked on: Rust 1.47
+-->
+
+It most recently worked on: <!-- version -->
+
+### Version with regression
+
+<!--
+Provide the version you are using that has the regression.
+-->
+
+`rustc --version --verbose`:
+```
+<version>
+```
+
+<!--
+Did the compiler crash? If so, please provide a backtrace.
+-->
+
+### Backtrace
+<!--
+Include a backtrace in the code block by setting `RUST_BACKTRACE=1` in your
+environment. E.g. `RUST_BACKTRACE=1 cargo build`.
+-->
+<details><summary>Backtrace</summary>
+<p>
+
+```
+<backtrace>
+```
+
+</p>
+</details>
+
+<!--
+Please change `nightly` to the release channel you found this regression in (stable, beta, or nightly).
+-->
+
+@rustbot modify labels: regression-from-stable-to-nightly
```

Comment:
Hm, I wonder if this will turn out to be a bad idea. Maybe we can have the default be something like "regression-untriaged" (I can create the label).

I worry that we'll otherwise get people filing nightly regressions, which are actually beta regressions.

Reply:
> Maybe we can have the default be something like "regression-untriaged" (I can create the label).

That seems like a good idea 👍 

> I worry that we'll otherwise get people filing nightly regressions, which are actually beta regressions.

Well it would probably be discovered when it's triaged, but yeah it probably makes sense to default to `regression-untriaged` so there's no confusion.

Reply:
Maybe we have the default be `regression-untriaged` and then say in a comment that you can change it if you know where it occurred?

Reply:
Yes, that seems reasonable. If it really becomes a problem we'll just remove the second bit.

Reply:
By the way, I've noticed as @Mark-Simulacrum points out that from time to time we've things tagged as nightly regressions that are beta regressions. As part of @rust-lang/wg-prioritization effort, I've been checking the ones that we track for the compiler weekly meeting.

I wonder if we could make a process on Triage WG and/or Prioritization WG and even automate some stuff to more actively check that the regressions are correctly labelled.